examples/burial-form/utils.ts

```diff
@@ -7,3 +7,19 @@ export const isBeforeDate = (
 ): () => string => {
   return () => isBefore(new Date(dateOne), new Date(dateTwo)) ? validationMessage : '';
 }
+
+export const stringToBoolean = (value: string): boolean => {
```

Comment:
In the branch this doesn't seem to be called anywhere, do we need this at all with this fix?

Reply:
Actually, we'll be using this method to transform the data while submit.